com.unity.render-pipelines.universal/Shaders/BakedLit.shader

```diff
@@ -146,7 +146,7 @@ Shader "Universal Render Pipeline/Baked Lit"
                 half3 normalWS = input.normalWS;
     #endif
                 normalWS = NormalizeNormalPerPixel(normalWS);
-                color *= SAMPLE_GI(input.lightmapUV, input.vertexSH, normalWS);
+                color *= SAMPLE_GI(input.staticLightmapUV, NOT_USED, input.vertexSH, normalWS);
```

Comment:
A dummy variable to show that the argument doesn't exist in this variant and to make sure the code fails compilation instead of silently introducing a bug.

Reply:
The dummy variable was removed again here https://github.com/Unity-Technologies/Graphics/pull/3386/commits/90a0a389dc505ca53e99fee8644b8c01633355f7